src/main/kotlin/no/nav/dagpenger/regel/grunnlag/Grunnlag.kt

```diff
@@ -56,6 +57,8 @@ class Grunnlag(
             grunnlagsBeregninger.map { beregning -> beregning.calculate(fakta) }.toSet().finnHøyesteAvkortetVerdi()
```

Comment:
`sjekkGyldigResultat` kan kanskje erstattes med en `.takeIf { 0.toBigDecimal() <= it?.uavkortet }` og gjenbruke `NoResultException `

```suggestion
            grunnlagsBeregninger.map { beregning -> beregning.calculate(fakta) }.toSet().finnHøyesteAvkortetVerdi().takeIf { 0.toBigDecimal() <= it?.uavkortet }
                ?: throw NoResultException("Ingen resultat for grunnlagsberegning")
```